lib/utils/device/ssh.ts

```diff
@@ -49,7 +49,7 @@ export async function performLocalDeviceSSH(
 			port: 2375,
 		});
 
-		const regex = new RegExp(`\\/?${escapeRegExp(opts.service)}_\\d+_\\d+`);
+		const regex = new RegExp(`(^|\\/)${escapeRegExp(opts.service)}_\\d+_\\d+`);
```

Comment:
This is the key fix. Previously, for a command like:
```
balena ssh 192.168.1.36 foo
```
... the regex would match for containers named `/freefoo_1_1` and `/foo_2_1` (services named "freefoo" and "foo"), but we want it to match the latter only.

**While my tests were successful for a recent version of balenaOS and balenaEngine, I don't know if different / older versions would return container names in different formats and cause the regex to fail, in particular the caret character to indicate "beginning of string".** The previous regex had a question mark that suggests that the forward slash is optional. Was the forward slash absent for other versions of balenaEngine? If it was absent, was there anything else in its place?  Reviewer feedback is welcome - @CameronDiver ? :-)